### PR TITLE
Add the script to update interface and update interface.

### PR DESCRIPTION
**Pull request by Shylock-Hg**

### CODE REVIEW

File: gen-interface.sh [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+for mod in common meta storage graph; do
+    wget https://raw.githubusercontent.com/vesoft-inc/nebula-common/master/src/common/interface/$mod.thrift
+done
+
+for mod in common meta storage graph; do
+    $NEBULA_THIRDPARYTY_HOME/2.0/bin/thrift1 --strict --allow-neg-enum-vals --gen "mstch_rust" -o ./src/interface/$mod/ $mod.thrift
+    mv ./src/interface/$mod/gen-rust/lib.rs ./src/interface/$mod
+    rmdir ./src/interface/$mod/gen-rust
+done
```
Comment (laura-ding) [inline gen-interface.sh:32]: These can be combined into a cycle.

File: gen-interface.sh [hunk 2]
```diff
@@ -0,0 +1,34 @@
+#! /usr/bin/env bash
+
+set -e
+
+NEBULA_THIRDPARYTY_HOME=
```
Comment (laura-ding) [inline gen-interface.sh:5]: You can add the link that how to install third-party.

Comment (Shylock-Hg) [reply gen-interface.sh:5]: The readme has note it.

File: gen-interface.sh (hunk 1 above)
Comment (Shylock-Hg) [reply gen-interface.sh:32]: The thrift file will include others.

**Review by Aiee — APPROVED** [review]

**Pull request merged**